src/test/java/com/transferwise/openbanking/client/test/TestAspspDetailsWithDifferentTrustAnchor.java

```diff
@@ -0,0 +1,70 @@
+package com.transferwise.openbanking.client.test;
+
+import com.transferwise.openbanking.client.configuration.AspspDetails;
+import com.transferwise.openbanking.client.configuration.SoftwareStatementDetails;
+import com.transferwise.openbanking.client.oauth.ClientAuthenticationMethod;
+import com.transferwise.openbanking.client.oauth.domain.GrantType;
+import com.transferwise.openbanking.client.oauth.domain.ResponseType;
+import com.transferwise.openbanking.client.oauth.domain.Scope;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+import java.util.Set;
+
+@Data
+@Builder
+public class TestAspspDetailsWithDifferentTrustAnchor implements AspspDetails {
```

Comment:
Why not implementing `getTrustAnchor()` in the already existing `com.transferwise.openbanking.client.test.TestAspspDetails`?

Reply:
I thought it was better to leave TestAspspDetails without the implementation of the method to prove that it was non-breaking and if not implemented then the default trust anchor would be used. I created TestAspspDetailsWithDifferentTrustAnchor with the implementation in place and added a test to ensure the non-default value was used